libs/core/src/lib/overflow-layout/overflow-layout.component.ts

```diff
@@ -309,17 +326,46 @@ export class OverflowLayoutComponent implements OnInit, AfterViewInit, OnDestroy
                             .toArray()
                             .findIndex((item) => item.focusable && item.elementRef.nativeElement === event.target);
                         if (index !== -1) {
-                            this._overflowLayoutService._keyboardEventsManager.setActiveItem(index);
+                            this._keyboardEventsManager?.setActiveItem(index);
                         }
                     }
 
                     if (KeyUtil.isKeyCode(event, [DOWN_ARROW, UP_ARROW, LEFT_ARROW, RIGHT_ARROW])) {
                         event.preventDefault();
 
                         // passing the event to key manager so, we get a change fired
-                        this._overflowLayoutService._keyboardEventsManager.onKeydown(event);
+                        this._keyboardEventsManager?.onKeydown(event);
                     }
                 })
         );
     }
+
+    /** @hidden */
+    private _setFocusKeyManager(): void {
+        if (!this.enableKeyboardNavigation) {
```

Comment:
can this be done with focusable-list cdk thing? probably not in scope of this pr

Reply:
I would suggest other PR for that